services/storage/prepare.mk

```diff
@@ -0,0 +1,2 @@
+prepare.storage:
+	@./bin/addCert.sh
```

Comment:
I don't think it's a good way to silently add some certs to the trusted list. Also, you don't provide any revocation of that trust on dev-env destroy.